plugins/opener/src/open.rs

```diff
@@ -0,0 +1,166 @@
+// Copyright 2019-2023 Tauri Programme within The Commons Conservancy
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-License-Identifier: MIT
+
+//! Types and functions related to shell.
+
+use serde::{Deserialize, Deserializer};
+
+use std::{ffi::OsStr, fmt::Display, path::Path, str::FromStr};
+
+/// Program to use on the [`open()`] call.
+#[derive(Debug)]
+pub enum Program {
+    /// Use the `open` program.
+    Open,
+    /// Use the `start` program.
+    Start,
```

Comment:
I can't remember what do `open` and `start` actually do? They don't seem to work on Windows at least

Reply:
I don't know tbh